be/src/exec/olap_scan_node.cpp

```diff
@@ -1038,6 +1038,7 @@ Status OlapScanNode::normalize_binary_predicate(SlotDescriptor* slot, ColumnValu
 
 void OlapScanNode::transfer_thread(RuntimeState* state) {
     // scanner open pushdown to scanThread
+    state->resource_pool()->acquire_thread_token();
```

Comment:
Can you tell the side effect of this operation? Will this thread hang if there is not enough thread resource?

Reply:
Add this operation just to Make AverageThreadTokens more accurate
   - AverageThreadTokens: 1.00  

Do not warry thread hang on this operation。
acquire_thread_token() just do a lightweight add operation without doing the really thread assignment 

inline void ThreadResourceMgr::ResourcePool::acquire_thread_token() {
    __sync_fetch_and_add(&_num_threads, 1);
}

@imay